build.gradle

```diff
@@ -0,0 +1,295 @@
+buildscript {
+  repositories {
+    maven {
+      url 'https://plugins.gradle.org/m2/'
+    }
+  }
+  dependencies {
+    // check updates with https://plugins.gradle.org/
```

Comment:
AFAIK https://github.com/ben-manes/gradle-versions-plugin can be used to check plugin updates.